File: frontend/doConnect/src/app/objects/question.ts
export class Question{
    id!:number;
    username!:string;
    topic!:string;
    description!:string;

    constructor(id:number, username:string, topic:string, description:string){

        this.id = id;
        this.description = description;
        this.topic = topic;
        this.username = username;

    }

}

File: backend/doConnect-QuestionAnswer/src/main/java/com/aakash/org/entity/Question.java
package com.aakash.org.entity;

/*
 * @Author - Aakash Verma
 */

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Entity
@Table(name = "Question")
public class Question {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	
	@OneToOne
	private User user;
	
	private String topic;
	
	private String description;
	
	private Boolean isApproved;
	
}
